packages/account/src/Components/account-limits/account-limits-turnover-limit-row.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { formatMoney } from '@deriv/shared';
+import AccountLimitsTableCell from './account-limits-table-cell';
+import AccountLimitsContext, { TAccountLimitsContext } from './account-limits-context';
+
+type TAccountLimitsTurnoverLimitRow = {
+    collection: Array<{
+        name: string;
+        payout_limit: number;
+        profile_name: string;
+        turnover_limit: number;
+    }>;
```

Comment:
```suggestion
    collection: TAccountLimits[];
```
Refactor this to its own type. A good suggested name would be `TAccountLimits`

```
TAccountLimits = {
        name: string;
        payout_limit: number;
        profile_name: string;
        turnover_limit: number;
}
```